### PR TITLE
SDK Flavor string translatable

### PR DESCRIPTION
**Pull request by stief510**
We added locales and as a result the flavor string needs to be translatable=false

### CODE REVIEW

File: android/src/main/res/values/appboy.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <!--- Internal Appboy Usage -->
-  <string name="com_appboy_sdk_flavor">REACT</string>
+  <string name="com_appboy_sdk_flavor" translatable="false">REACT</string>
```
Comment (Bucimis) [inline android/src/main/res/values/appboy.xml]: lets put the translatable attribute before the name attribute to pattern match what we do elsewhere (e.g in AppboyProject)

**Comment by stief510** [comment]
I can switch that around if you would prefer

**Comment by Bucimis** [comment]
@stief510 that would be great! thanks

**Comment by stief510** [comment]
No problem. Done.

**Comment by Bucimis** [comment]
@stief510 looks great! thanks for adding this - will merge now :)

**Pull request merged**